graphqlapi/src/main/java/io/stargate/graphql/GraphqlActivator.java

```diff
@@ -42,7 +43,7 @@
   private ServicePointer<Metrics> metrics = ServicePointer.create(Metrics.class);
 
   @GuardedBy("this")
-  private WebImpl web;
+  private Server server;
```

Comment:
All that was left of `WebImpl` was just an empty shell around `Server`, I removed it.